benchexec/tablegenerator/react-table/src/components/LinkOverlay.js

```diff
@@ -96,7 +99,7 @@ export default class LinkOverlay extends React.Component {
           const content = await response.text();
           this.setState({ content });
         } else {
-          this.loadFileFromZip(url);
+          throw Error(`Received response status ${response.status}`);
```

Comment:
There is no actual change in functionality here. The statement is within a `try` block, that calls the `loadFileFromZip` function when an error occurs - and when entering this else block, the response has returned an error (or at least that something isn't working). So I thought it would be more intuitive to also throw an error in this case and call the `loadFileFromZip` function from the `except` block. Also this helped me a bit during debugging, because there was only a single entry for this function with different errors, showing why exactly the previous attempt failed.

Reply:
Nice!